ocfweb/docs/templates/officers.html

```diff
@@ -29,26 +23,24 @@
           href="/about/staff">weekly meetings</a>.
         </p>
       <h3>
-        General managers
-        &lt;<a href="mailto:gm@ocf.berkeley.edu">gm@ocf.berkeley.edu</a>&gt;
```

Comment:
I hate these emails. One would email gm@ocf thinking they are emailing only the GM, but unwittingly also email 10 other officers.